src/ucp/core/ucp_rkey.c

```diff
@@ -349,7 +349,7 @@ UCS_PROFILE_FUNC(ucs_status_t, ucp_ep_rkey_unpack_internal,
      * allocations are done from a memory pool.
      * We keep all of them to handle a future transport switch.
      */
-    if (md_map <= UCS_BIT(UCP_RKEY_MPOOL_MAX_MD)) {
+    if (ucs_popcount(md_map) <= UCP_RKEY_MPOOL_MAX_MD) {
```

Comment:
cache popcount, to not recalc in line 357

Reply:
fixed